site/book/06-deploying-packages/02-applying-a-package.md

```diff
@@ -0,0 +1,102 @@
+Once you have initialized the package, you can deploy it using `live apply`:
+
+```shell
+$ kpt live apply wordpress
+installing inventory ResourceGroup CRD.
+service/wordpress created
+service/wordpress-mysql created
+deployment.apps/wordpress created
+deployment.apps/wordpress-mysql created
+persistentvolumeclaim/mysql-pv-claim created
+persistentvolumeclaim/wp-pv-claim created
+6 resource(s) applied. 6 created, 0 unchanged, 0 configured, 0 failed
+0 resource(s) pruned, 0 skipped, 0 failed
+```
+
+Alternatively, you can specify whether you want to wait for resources to be
+reconciled:
+
+```shell
+$ kpt live apply wordpress --reconcile-timeout=2m
+```
+
+?> Refer to the [apply command reference][apply-doc] for usage.
+
+## `ResourceGroup` CRD
+
+By default, `live apply` automatically installs the `ResourceGroup` CRD (unless
+`--dry-run` is specified) since it needs to create the associated
+`ResourceGroup` custom resource. You can also manually install the CRD before
+running `live apply`:
+
+```shell
+$ kpt live install-resource-group
+```
+
+?> Installing this CRD requires sufficient ClusterRole permission, so you may
+need to ask your cluster admin to install it for you.
+
+## Server-side vs Client-side apply
+
+By default, `live apply` command uses client-side apply. The updates are
+accomplished by calculating and sending a patch from the client. Server-side
+apply, which can be enabled with the `--server-side` flag, sends the entire
+resource to the server for the update.
+
+## Dry-run
+
+You can uses the `--dry-run` flag to get break down of operations that will be
```

Comment:
I think we should also mention validation here. The main advantage of dry-run (in particular server-side dry-run) is that the resources pass through all the validation steps so any errors can be caught early and before any changes have been made to the live state.

Reply:
Yeah, validation errors will be caught during apply, but only when the resource in question is being applied to the cluster. Catching validation errors at this point will likely lead to the package being only partially applied.